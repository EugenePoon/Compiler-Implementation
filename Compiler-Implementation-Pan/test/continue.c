int main()
{
    int i;
    for(i = 0; i < 13; i++) {
        if(i % 2 == 1)
            continue;
        print i;
    }
}