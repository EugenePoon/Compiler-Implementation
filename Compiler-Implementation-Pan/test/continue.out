24 19 0 5 25 15 1 13 0 0 12 15 -1 16 43 13 11 0 2 5 0 1 6 17 27 16 32 13 11 22 15 -1 13 11 13 13 11 0 1 1 12 15 -2 13 11 0 13 7 18 15 21 0